### PR TITLE
Long commit msg

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @jamieburkedev
- [ ] @Michaeljivan
- [ ] @MattRenda
- [ ] @tweetyly
- [ ] @gurdevhora

### PR DESCRIPTION
**Pull request by ericalloyd**
Additional details around pull request.
Testing instructions.
Anything a reviewer would need in order to understand your changes.

Resolves #2

### CODE REVIEW

File: _posts/0000-01-07-ericalloyd.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I love bright colors!!
```
Comment (MattRenda) [inline _posts/0000-01-07-ericalloyd.md:6]: Nice caption!

Comment (MattRenda) [reply _posts/0000-01-07-ericalloyd.md:6]:
```suggestion
I love bright colors!!!!!!!!!!!!!!!!
```

Comment (MattRenda) [reply _posts/0000-01-07-ericalloyd.md:6]: Now people really know you like bright colors.

Comment (ronanmcallister) [inline _posts/0000-01-07-ericalloyd.md:6]: Code review

Comment (ronanmcallister) [reply _posts/0000-01-07-ericalloyd.md:6]: Not sure I did this code review correctly.  OK/Approved

**Pull request merged**